src/db_rows.rs

```diff
@@ -130,6 +130,28 @@ impl NftMetadata {
     // fn from_blob(blob: serde_json::Value) -> Option<Self> {}
 }
 
+#[derive(Clone, diesel::Insertable, diesel::AsChangeset)]
+#[table_name = "nft_attributes"]
+pub struct NftAttributes {
+    pub nft_metadata_id: String,
+    pub nft_contract_id: String,
+    pub attribute_type: String,
+    pub attribute_value: Option<String>,
```

Comment:
Why is this different from the table def?

Reply:
Lost change, resolved